js/web/lib/wasm/jsep/webgpu/ops/conv-transpose.ts

```diff
@@ -226,12 +228,64 @@ const createConvTranspose2DProgramInfoLoader =
       };
     };
 
+// for transposing weight tensor from [C, M/group, KH, KW] to [KH, KW, M/group, C]
+const weightTransposeAttribute: TransposeAttributes = createAttributeWithCacheKey({perm: [2, 3, 1, 0]});
+
 const convTranspose2d =
     (context: ComputeContext, inputs: readonly TensorView[], attributes: ConvTransposeAttributes): void => {
       const adjustedAttributes = getAdjustedConvTransposeAttributes(attributes, inputs);
+      const isChannelsLast = attributes.format === 'NHWC';
+      const hasBias = inputs.length === 3;
+      if (adjustedAttributes.group !== 1 || attributes.strides.reduce((a, b) => a * b, 1) !== 1) {
```

Comment:
Why choose the naive method when the strides size is larger than 1?

Reply:
Removed condition on strides. However dilations > 1 causing errors. So, modified code to use naive implementation in case of dilation > 1.

Reply:
Removed the condition limiting dilations to 1 to use matmul implementation.